.gitattributes

```
@@ -0,0 +1,2 @@
+ui/packages/shared/client/src/parca/ linguist-generated
+ui/packages/shared/client/src/google/ linguist-generated
```

```suggestion
ui/packages/shared/client/src linguist-generated
gen linguist-generated
```
this also :point_up: